cdci_data_analysis/analysis/job_manager.py

```diff
@@ -186,8 +186,11 @@ def write_dataserver_status(self, status_dictionary_value=None,
         if status_dictionary_value is None:
             pass
         else:
-            self.monitor['status'] = status_dictionary_value
-
+            if status_dictionary_value in self._allowed_job_status_values_:                
+                self.monitor['status'] = status_dictionary_value
+            else:
+                self.monitor['status'] = "progress" # any unknown message is progress
```

Comment:
This will divert any unknown status to `progress`, right? Would not be useful to issue a warning that an unknown status has been returned?

Reply:
I thought about it but it would produce too many messages. The callbacks were made to provide a lot of auxiliary information in anticipation of several future features, this includes other statuses. So regulating their content from dispatcher too much is not ideal.
I added debug.